src/fides/api/models/privacy_experience.py

```diff
@@ -615,6 +636,12 @@ def cache_saved_and_served_on_consent_record(
     if not fides_user_provided_identity:
         return
 
+    if not record_type:
+        # Some elements of the TCF form, like vendor relationships don't have saved preferences
+        return
+
+    assert not isinstance(consent_record, TCFVendorRelationships)  # For mypy
+
```

Comment:
One of the sections in the TCF Experience, the Vendor Relationships, section cannot have preferences saved against it.  This is a display-only section, for the special features, special purposes, and features that show up in the Vendors tab.  When we're retrieving previously saved preferences for the other sections, I skip this one.

Reply:
nice, makes sense!